scripts/peek_dma.py

```diff
@@ -0,0 +1,50 @@
+def main():
```

Comment:
Well, these scripts aren't stand alone Python programs, they're intended to run within the environment of the debug console.  Putting them in utils doesn't seem right, because that directory tends to hold stand alone utilities.
We actually might want to break the console debugger out into it's own sub-directory rather then put it in utils.  That would make it easier to split into multiple py files.  If we did that we could move the scripts directory under there, maybe a scripts sub-directory under the console debug directory.
I don't feel strongly about this so if you want to just put everything in utils we can do that.  I think we'll probably end up collecting a lot of handy little scripts like this for all sorts of things though, so it might start to get cluttered.

Reply:
OK, do however you're comfortable.  I can clean it up later when I'm done with the debug patch.